koku/reporting/migrations/0102_ocp_all_materialized_views.py

```diff
@@ -0,0 +1,465 @@
+# Generated by Django 2.2.10 on 2020-03-09 22:04
+from django.contrib.postgres.indexes import GinIndex
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
I'm not seeing the create models migrations here. How are you making migrations with Django? Those should have been automatically generated.

Reply:
The models were created by hand as unmanaged. That is the pattern I saw for the mat-view stuff I looked over. Was this incorrect?

Reply:
That is correct, but I believe Django will still create a `migrations.CreateModel` in a migration even with managed=False. I think that keeps it flagged as a model that has already been migrated. Was wondering if you were manually writing migrations, or using `make make-migrations` which uses the Django manage command to auto-generate migrations (that you can then add RunSQLs to).